form.yml

```diff
@@ -90,6 +91,24 @@ attributes:
         data-option-for-cluster-pitzer: false,
         data-option-for-cluster-ascend: false,
         ]
+  python_version:
+    widget: "select"
+    label: "Python Version"
+    help: |
+      Specify the Python version to match the one used in your Conda environment. This ensures compatibility when setting up the Spark cluster and helps prevent issues related to Python version mismatches. Choose Python 3.12 if you want to use the default PySpark kernel.
+    options:
+      - [
+          "3.12", "python/3.12",
+           data-option-for-cluster-pitzer: false,
+        ]
+      - [
+          "3.10", "miniconda3/24.1.2-py310",
```

Comment:
I don't know if this will work, but `auto_modules_python` will automatically populate this drop down based on the module spider output. That is, it'll automatically update itself when we add new versions.

That said - given this module is not `python`, instead it's `miniconda3` I'm not sure if we can go that route. 

Can `auto_modules_python` work here?

Reply:
I am not aware of `auto_modules_python`. Based on your explanation, I suspect it might work with the miniconda3 module, especially since its version format is not standard (e.g., `miniconda3/24.1.2-py310`). Also, will `auto_modules_python`  generate a list of available modules, or does it generate a list of version strings?

Reply:
We can proceed with `auto_modules_python`. This is a great feature that helps reduce maintenance efforts on both sides.

Could you please apply the changes using auto_modules_python?
Also, please update the help message to:
```
Specify the Python version to match the one used in your Conda environment. This ensures compatibility when setting up the Spark cluster and helps prevent issues related to Python version mismatches.
```
Thank you!

Reply:
OK - I'll submit the pull request shortly.

These versions are available on Pitzer - do we need to filter any of them (I'm thinking specifically 2.7)
![image](https://github.com/user-attachments/assets/ecc7ab61-fba5-4e7f-9cf5-d900b5408c31)

Reply:
I have #42 open with the help message. I think pitzer is our only hanging thread - don't know what modules are going to be available when we upgrade to RHEL9 and/or if we need to filter any versions like 2.7 for example.

Reply:
For PItzer RHEL 7, can we disable this Python selection? We will install any Python version by request for Pitzer RHEL 9.

Reply:
> For PItzer RHEL 7, can we disable this Python selection? We will install any Python version by request for Pitzer RHEL 9.

Sure we can keep the hard coded value that we currently have for pitzer.